### ISSUE TITLE
No me too button on MantisBT 2.12.0 :-(

### ISSUE THREAD

**Issue by takenek** [mantisbt-plugins/GaugeSupport#3]
Hello
I just install Your addon and i see "Issue Support Ranking" but noone see "me too" button.
I check and in Community Support Ranking for issues configuration and i give proper permission but still nothing happened.

I give in configuration:
Statusses to exclude from overview: Closed
Severities to include in overview: i select all
Resolutions to exclude from overview: Copy

And still no "me too" button for anyone....
Can you take a look on this please?

Best Regards
TaKeN

**Comment by RedSpid3r** [comment]
yes, this does not seem to work on 2.x mantis :(

**Comment by ghost** [comment]
​It does work, check it out here:
http://www.nuy.info/mantis2
Unfortunately I cannot reproduce your problem.
I installed on a fresh system and it works like a charm.

Op wo, 04/04/2018 05:21 PM, RedSpid3r <notifications@github.com> schreef:
> 
yes, this does not seem to work on 2.x mantis :(
—
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.




  
 *

**Comment by takenek** [comment]
Hello CasN - i just create account on Your mantis and this is old version 2.1.0 and fresh is 2.13.1.
And i don`t see on Your mantis "me too" button too i just check all open tickets.... and i see only "Assign to", "clone"  and "Monitor" there is no "Me too" button.

Best Regards
TaKeN

**Comment by fanjing-whu** [comment]
> Hello CasN - i just create account on Your mantis and this is old version 2.1.0 and fresh is 2.13.1.
> And i don`t see on Your mantis "me too" button too i just check all open tickets.... and i see only "Assign to", "clone"  and "Monitor" there is no "Me too" button.
> Best Regards
> TaKeN

there are some code missing in gauge_form.php
```
	$current= explode(",", plugin_config_get( 'gaugesupport_excl_resolution' ));
	if(in_array(bug_get_field($bugid, 'resolution'),$current)){
		return;
	}
	
	$current= explode(",", plugin_config_get( 'gaugesupport_incl_severity' ));
	if(!in_array(bug_get_field($bugid, 'severity'),$current)){
		return;
	}

	$current= explode(",", plugin_config_get( 'gaugesupport_excl_status' ));
	if(in_array(bug_get_field($bugid, 'status'),$current)){
		return;
	}
```
adding above code in gauge_form.php may help(read the code, you will find the position easily)

And, this plugin use db_query_bound to query the database in gauge_form.php
however, 
```
/**
 * function alias for db_query() for legacy support of plugins
 * @deprecated db_query should be used in preference to this function. This function may be removed in 2.0
 */
```
if something error, change it to `db_query()`.

After that, the plugin content may show on the bottom

**Comment by takenek** [comment]
After add this code still no "me too" button.
Checked on version: 2.17.0

**Comment by ghost** [comment]
TakeN, what do you mean with the MeToo button?  Ido not have this option in my code at all, just a button which says "Declare your stance".

**Comment by ghost** [comment]
@Devilkuro, I indeed missed inclusion of severities, corrected in version 2.03. Allso corrected the db_query statemnet

**Comment by dregad** [comment]
> corrected in version 2.03

See my note in https://github.com/mantisbt-plugins/GaugeSupport/issues/2#issuecomment-428236728

**Comment by ghost** [comment]
Now updated the code here, the right place!
